wemake_python_styleguide/logic/tree/functions.py

```diff
@@ -103,3 +103,11 @@ def is_generator(node: AnyFunctionDef) -> bool:
         if is_contained(node=body_item, to_check=(Yield, YieldFrom)):
             return True
     return False
+
+
+def check_decorators(node: AnyFunctionDef, name: str) -> bool:
+    """Check if name is in decorator name."""
+    return any(
+        name in source.node_to_string(decorator)
```

Comment:
I also feel like this place can brake. Let's replace `in` with `==`

Reply:
Sure.

Reply:
`attribute.setter` will be the result from `node_to_string(decorator)` if the decorator is `@attribute.settter`. Since the word before the dot `.` can be whatever, it will be hard to find methods with property setter decorator.

Reply:
I think `in` should be correct since i have seen others use that for checking decorators:
```
_PROPERTY_EXCEPTIONS: Final = frozenset(('property', '.setter'))
...
def is_property_setter(node: ast.AST) -> bool:
    """Check that function decorated with ``@property.setter``."""
    if isinstance(node, FunctionNodes):
        for decorator in node.decorator_list:
            if node_to_string(decorator) in _PROPERTY_EXCEPTIONS:
                return True
    return False
```